systemd/ocp-cluster-ca.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -x
+
+source /usr/local/bin/crc-systemd-common.sh
+export KUBECONFIG="/opt/kubeconfig"
+
+wait_for_resource configmap
+
+custom_ca_path=/opt/crc/custom-ca.crt
+
+if [ ! -f ${custom_ca_path} ]; then
+    echo "Cert bundle /opt/crc/custom-ca.crt not found"
+    exit 1
+fi
+
+if [ ! -f /opt/crc/pass_kubeadmin ]; then
+    echo "kubeadmin password file not found"
+    exit 1
+fi
+
+PASS_KUBEADMIN="$(cat /opt/crc/pass_kubeadmin)"
+
+oc create configmap client-ca-custom -n openshift-config --from-file=ca-bundle.crt=${custom_ca_path}
+oc patch apiserver cluster --type=merge -p '{"spec": {"clientCA": {"name": "client-ca-custom"}}}'
+oc create configmap admin-kubeconfig-client-ca -n openshift-config --from-file=ca-bundle.crt=${custom_ca_path} \
+--dry-run -o yaml | oc replace -f -
```

Comment:
these steps comes from `crc-cloud` and based on this doc; https://access.redhat.com/solutions/6054981